.travis.yml

```diff
@@ -0,0 +1,24 @@
+dist: trusty
```

Comment:
`xenial` is the Travis default. Do we need to define `trusty` here`

Reply:
Removed `dist`.